backtrace-library/src/main/java/backtraceio/library/services/BacktraceReportSender.java

```diff
@@ -102,6 +112,80 @@ static BacktraceResult sendReport(String serverUrl, String json, List<String> at
         return result;
     }
 
+    /**
+     * Send HTTP request for certain url server with information about events
+     *
+     * @param serverUrl             server http address to which the request will be sent
+     * @param json                  message wih information about events
+     * @param payload               information about events
+     * @param errorCallback         event that will be executed after receiving an error from the server
+     * @return information from the server about the result of processing the request
+     */
+    public static EventsResult sendEvents(String serverUrl, String json, EventsPayload payload, OnServerErrorEventListener errorCallback) {
+        HttpURLConnection urlConnection = null;
+        EventsResult result;
+        int statusCode = -1;
+
+        try {
+            URL url = new URL(serverUrl);
+            urlConnection = (HttpURLConnection) url.openConnection();
+
+            urlConnection.setRequestMethod("POST");
+            urlConnection.setUseCaches(false);
+
+            urlConnection.setDoOutput(true);
+            urlConnection.setDoInput(true);
+
+            urlConnection.setChunkedStreamingMode(128 * 1024);
```

Comment:
Magic variable. Why 128k?

Reply:
@rqbacktrace if you want to reuse it extract it to const

I'm not sure why I've set it like this, probably I was experimenting with different values but I'm ok with changing that if you need